spectrum/autism_prevalence_map/views.py

```diff
@@ -49,7 +53,8 @@ def index(request):
                     "timeline_type": timeline_type, 
                     "meanincome": meanincome, 
                     "education": education,
-                    "last_updated_on": last_updated_on}
+                    "last_updated_on": last_updated_on,
+                    'environment': environment}
```

Comment:
maybe just put the conditional logic here to send `min` or not as `context_dict`, so that the view file does not have the logic?